http/http-minimum/src/test/java/io/quarkus/ts/http/minimum/DevModeHttpMinimumIT.java

```diff
@@ -24,22 +25,19 @@
     @DevModeQuarkusApplication
     static DevModeQuarkusService app = new DevModeQuarkusService();
 
+    @Disabled("TODO: Fix flaky test: https://github.com/quarkus-qe/quarkus-test-framework/issues/146")
```

Comment:
Commits https://github.com/quarkus-qe/quarkus-test-suite/pull/72/commits/4911e0536e736952e4707bacbfe3a9513ed90966 and https://github.com/quarkus-qe/quarkus-test-suite/pull/72/commits/e782a626bc4c7bfa00083567583351598b95944f are reverts to each other, the only relevant part is `@Disabled("TODO: Fix flaky test: https://github.com/quarkus-qe/quarkus-test-framework/issues/146")`

Please squash them.

Reply:
Thanks for spotting! Fixed now.